xbmc/messaging/helpers/DialogHelper.h

```diff
@@ -74,6 +76,27 @@ DialogResponse ShowYesNoDialogText(CVariant heading, CVariant text, CVariant noL
                                    CVariant yesLabel = CVariant(), uint32_t autoCloseTimeout = 0);
 
 /*!
+\brief This is a helper method to send a threadmessage to open a Yes/No dialog box with a cutom button
+
+\param[in]  heading           The text to display as the dialog box header
+\param[in]  text              The text to display in the dialog body
+\param[in]  noLabel           The text to display on the No button
+                              defaults to No
+\param[in]  yesLabel          The text to display on the Yes button
+                              defaults to Yes
+\param[in]  customLabel       The text to display on the optional 3rd custom button
+                              defaults to empty and button not shown
+\param[in]  autoCloseTimeout  The time before the dialog closes
+                              defaults to 0 show indefinitely
+\return -1 on cancelled, 0 on no, 1 on yes and 2 on 3rd custom response
+\sa ShowYesNoDialogLines
+\sa CGUIDialogYesNo::ShowAndGetInput
+\sa DialogYesNoMessage
+*/
+DialogResponse ShowYesNoOtherDialogText(CVariant heading, CVariant text, CVariant noLabel = CVariant(), CVariant yesLabel = CVariant(),
```

Comment:
I would suggest `ShowYesNoCustomDialog`. Using the dialog with lines should be deprecated IMO.